.dev/gettingStarted.R

```diff
@@ -0,0 +1,45 @@
+#### Getting started with VEuPathDB package development
+
+### Setting up the environment
+
+## Check out where we are (assuming running the docker container and that plot.data and microbiomeComputations also checked out)
+list.dirs(".", recursive=F)
+# [1] "./.devcontainer"          "./microbiomeComputations"
+# [3] "./plot.data"           "./veupathUtils"
+
+## Set directory
+# Often it's easiest to set the directory to the repo in which we're actively developing
+setwd('veupathUtils') # If developing veupathUtils
+
+## For developing veupathUtils
+# TODO
+
+## For developing plot.data
+# First have plot.data checked out in a directory adjacent to veupathUtils
+# Handle plot.data environment (time zone, for example)
+library(dotenv)
+load_dot_env(file=".dev/.env")
```

Comment:
im a little worried someone might think this is a script they could run or something.. maybe make clearer this files intent somehow